spec/javascripts/controllers/inventories/AddInventoryUsersCtrlSpec.js

```diff
@@ -0,0 +1,52 @@
+(function() {
+  'use strict';
+  describe('Controller: AddInventoryUsersCtrl', function() {
```

Comment:
Style:  You're describing a controller in your test, so there's no need to suffix it with `Ctrl` here.